MediaPlayer.py

```diff
@@ -0,0 +1,55 @@
+import re
+from pygame import mixer
+import os
+
+
+# p = "C:\Users\Mulat\Desktop\MediaPlayerProject\songs"
+
+
+def load_song_dir_to_dictionary() -> dict:
+    user_path = input()
+    songs_name = os.listdir(user_path)
+    files_path = [os.path.abspath(user_path) for x in os.listdir(user_path)]
+    for i in range(0, len(files_path)):
+        files_path[i] = files_path[i] + os.sep + songs_name[i]
+    for i in range(0, len(songs_name)):
+        songs_name[i] = make_file_pretty_name(songs_name[i])
+    return dict(zip(songs_name, files_path))
+
+
+def make_file_pretty_name(song_name):
+    string_array = re.split('[-_ ]', song_name)
+    return string_array[0] + " " + string_array[1]
```

Comment:
There are a few cases where this function won't behave well. This is a nice candidate for testing because it's pure functionality - for now try adding a few `assert` statements right below the function